examples/sso/variables.tf

```diff
@@ -17,6 +17,12 @@ variable "region" {
   description = "The AWS Region"
 }
 
+
+variable "aws_sso_region" {
```

Comment:
I would move this out of the tfscaffold variables section into the granted-specific section.